meta/netbeansrelease.json

```diff
@@ -441,6 +441,34 @@
             "year": "2021"
         }
     },
+    "release1243": {
+        "position": "12",
+        "ant": "ant_latest",
+        "jdk": "jdk_1.8_latest",
+        "jdk_apidoc": "https://docs.oracle.com/javase/8/docs/api/",
+        "maven": "maven_3.3.9",
+        "versionName": "12.4.301",
+        "tlp": "true",
+        "apidocurl": "https://bits.netbeans.org/12.4/javadoc",
+        "update_url": "https://netbeans.apache.org/nb/updates/12.4/updates.xml.gz?{$netbeans.hash.code}",
+        "plugin_url": "https://netbeans.apache.org/nb/plugins/12.4/catalog.xml.gz",
+        "milestones": {
+            "d26abee5706b6a1c7cd8ae6589d059b45144576a": {
+                "version": "vc1",
```

Comment:
You probably don't want `vc1` in the version field. If this is really intended to be voted on, just leave version out.  Only use version for `beta` or `rc`. Can use `"vote" : "1"` but that doesn't really do a lot.

Reply:
done in 61adcff